doc/source/examples/analysis/custom_parallel_analysis.ipynb

```diff
@@ -0,0 +1,986 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "result = np.asarray(result).T\n",
+    "\n",
+    "labels = ['all', 'x-axis', 'y-axis', 'z-axis']\n",
+    "for col, label in zip(result, labels):\n",
+    "    plt.plot(col, label=label)\n",
+    "plt.legend()\n",
+    "plt.ylabel('Radius of gyration (Å)')\n",
+    "plt.xlabel('Frame');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Parallelization with dask\n",
+    "\n",
+    "[Dask](https://docs.dask.org/en/latest/) is a flexible library for parallel computing in Python. It provides advanced parallelism for analytics that has been integrated or utilized in many scientific softwares. It can be scaled from one single computer to a cluster of computers inside a HPC center.\n",
+    "\n",
+    "It has a dynamic task scheduling system that can be used either in synchronous (single-core), threading, multiprocessing or distributed cluster. The wrapping function in dask, `dask.delayed`, mimics for loops and wraps custom normal python code into a dask graph. Then it can be easily run in parallel, and visualized by `dask.visualize()` to examine if the task is well distributed. To be more precise, the code inside`dask.delayed` will not run immediately but be pushed into a job queue waiting for submission (`compute()` or `persist()`). A more complex dependency can also be built among different jobs, which might be useful for a complex analysis.\n",
```

Comment:
```suggestion
    "Dask has a dynamic task scheduling system with synchronous (single-threaded), threaded, multiprocessing and distributed schedulers. The wrapping function in dask, `dask.delayed`, mimics for loops and wraps Python code into a Dask graph. This code can then be easily run in parallel, and visualized with `dask.visualize()` to examine if the task is well distributed. The code inside `dask.delayed` is not run immediately on execution, but pushed into a job queue waiting for submission. These queues can be resolved or executed with commands like `compute()` or `persist()`. A more complex dependency can also be built among different jobs, which might be useful for a complex analysis.\n",
```

I find this very short explanation a bit confusing, especially the bits at the end ("A more complex dependency..." expecially. Could you go into more detail or just link to Dask resources explaining these concepts?